pkgs/applications/science/electronics/kicad/versions.nix

```diff
@@ -27,25 +27,25 @@
   };
   "kicad-unstable" = {
     kicadVersion = {
-      version =			"2020-02-10";
+      version =			"2020-03-16";
```

Comment:
According to the guidelines, the commit message should be "kicad-unstable: 2020-02-10 -> 2020-03-16"

Reply:
right!